src/lowlevel.ts

```diff
@@ -252,10 +252,10 @@ function catchNotFound(err: unknown): [] {
 export async function queryRaw<T = unknown>(
   config: ReadConfig,
   statement: string,
-  opts: Signal = {}
+  signal?: Signal
```

Comment:
Renamed. As mentioned above for controller, this isn't really an "options" object.

Reply:
We left it as options in case we wanted to add additional opts to it at a later date.

Reply:
I see. So you would have unioned `Signal` with some other type in the future? Are you suggesting I change it back?

Reply:
Yes we could have, but naw it is probably fine. Worst case is that we have to change the parameter name in the future. But since it is already an _object_, it'll be a pretty safe change.